orchagent/pfc_detect_mellanox.lua

```diff
@@ -132,8 +124,8 @@ for i = n, 1, -1 do
                                 local pfc_rx_packets_string = '"pfc_rx_packets","' .. tostring(pfc_rx_packets) .. '","pfc_rx_packets_last","' .. tostring(pfc_rx_packets_last) .. '",'
                                 local storm_condition_string = '"pfc_duration","' .. tostring(pfc_duration) .. '","pfc_duration_last","' .. tostring(pfc_duration_last) .. '",'
                                 local timestamps = '"timestamp","' .. timestamp_string .. '","timestamp_last","' .. timestamp_last .. '","effective_poll_time","' .. effective_poll_time .. '"'
-                                if effective_poll_time_lasttime ~= false then
-                                    timestamps = timestamps .. ',"effective_pfcwd_poll_time_last","' .. effective_poll_time_lasttime .. '"'
+                                if global_effective_poll_time_lasttime ~= false then
```

Comment:
@stephenxs global_effective_pfcwd_poll_time_last is the variable that gets the timestamp, right? why are we comparing with boolean(false)? Pls check

Reply:
Not really 
It's timestamp if it succeeds to fetch it from db
Or false if it fails to fetch it from db
Comparing it with false means to check whether it is in the db